app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayController.kt

```diff
@@ -18,8 +18,11 @@ import mozilla.components.concept.engine.mediasession.MediaSession.PlaybackState
 import mozilla.components.feature.tabs.TabsUseCases
 import mozilla.components.lib.state.DelicateAction
 import mozilla.telemetry.glean.private.NoExtras
+import org.mozilla.fenix.BrowserDirection
+import org.mozilla.fenix.GleanMetrics.Events
 import org.mozilla.fenix.GleanMetrics.Tab
```

Comment:
Let's alias this as `GleanTab` instead of adding `SyncTab`. It was weird seeing `Tab` above and then `SyncTab` here when I was pretty sure we didn't have a `SyncTab` object.